types/react-adaptive-hooks/memory.d.ts

```diff
@@ -0,0 +1,14 @@
+export interface MemoryStatus {
+    deviceMemory: number;
+    totalJSHeapSize: number | null;
```

Comment:
can these just be optional? `totalJSHeapSize?: number;`

Reply:
are they likely to ever be `null` explicitly, is what i mean

Reply:
They are explicitly set to `null` if not defined.